projects/lcms/cms_cgats_fuzzer.c

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2023 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <stdint.h>
+#include "lcms2.h"
+
+#define NPOINTS_IT8 10  // (17*17*17*17)
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+
+    if (size < 8){
+        return 0;
+    }
+
+    cmsContext context = cmsCreateContext(NULL, (void *)data);
+
+    uint32_t Row = *((uint32_t *)data);
+    uint32_t Col = *((uint32_t *)data+1);
+
+    /* Write */
+    cmsHANDLE  it8;
+    cmsInt32Number i;
+
+    it8 = cmsIT8Alloc(0);
+    if (it8 == NULL) return 0;
+
+    cmsIT8SetSheetType(it8, "LCMS/TESTING");
+    cmsIT8SetPropertyStr(it8, "ORIGINATOR",   "1 2 3 4");
+    cmsIT8SetPropertyUncooked(it8, "DESCRIPTOR",   "1234");
+    cmsIT8SetPropertyStr(it8, "MANUFACTURER", "3");
+    cmsIT8SetPropertyDbl(it8, "CREATED",     (data[0] % 256) / 255.0);
+    cmsIT8SetPropertyDbl(it8, "SERIAL",      (data[1] % 256) / 255.0);
+    cmsIT8SetPropertyHex(it8, "MATERIAL",     0x123);
+
+    cmsIT8SetPropertyDbl(it8, "NUMBER_OF_SETS", NPOINTS_IT8); // ?
```

Comment:
`// ?` is this something that is forgotten to be removed? also, am not quite sure I understand the comment in `#define NPOINTS_IT8 10  // (17*17*17*17)`

Reply:
`(17*17*17*17)` was taken from the project's file ./testbed/testcms2.c#6982, but we don't know how it's calculated, so all comments were removed.